src/main/java/dev/enginecode/eccommons/infrastructure/json/repository/mapping/Mapping.java

```diff
@@ -0,0 +1,5 @@
+package dev.enginecode.eccommons.infrastructure.json.repository.mapping;
+
+public interface Mapping {
```

Comment:
I think the name is too much general as the mappers are not only objects for json serialising/deserialising, but also mappers as for example design pattern to change one object into another. Could we modify it to something more precise, e.g. JsonConverter (JsonMapper is best but used by the implementation)

Reply:
OK, renamed to ObjectMapping - the main purpose of it is to map object to sth else by reading or writing methods.